tests/test_archiver.py

```diff
@@ -10,6 +10,7 @@
 class ArchiverTests(utils.ApiTest):
     @mock.patch("historical_system_profiles.archiver._check_and_send_notifications")
     def test_archive_profile(self, mock_check_and_send):
+        breakpoint()
```

Comment:
What is the purpose of having this here?

Reply:
That's for sure a debugging call to run an interactive console from inside of the test.

It should not be there for sure :-)

Reply:
my mistake... forgot to remove it